### PR TITLE
Improve sample app and add interaction to swap bottom view controller

### PR DESCRIPTION
**Pull request by felixLam**
simulator screen shot 30 06 2016 16 24 26
simulator screen shot 30 06 2016 16 24 32

### CODE REVIEW

File: ISHPullUpSample/ISHPullUpSample/BottomVC.swift [hunk 1]
```diff
@@ -38,6 +38,15 @@ class BottomVC: UIViewController, ISHPullUpSizingDelegate, ISHPullUpStateDelegat
         pullUpController.toggleState(animated: true)
     }
 
+    @IBAction func buttonTappedLearnMore(_ sender: AnyObject) {
```
Comment (shagedorn) [inline ISHPullUpSample/ISHPullUpSample/BottomVC.swift]: private

File: ISHPullUpSample/ISHPullUpSample/ContentVC.swift [hunk 2]
```diff
@@ -11,14 +11,16 @@ import ISHPullUp
 
 class ContentVC: UIViewController, ISHPullUpContentDelegate {
     @IBOutlet private weak var mapView: MKMapView!
+    @IBOutlet weak var layoutAnnotationLabel: UILabel!
```
Comment (shagedorn) [inline ISHPullUpSample/ISHPullUpSample/ContentVC.swift]: private

File: ISHPullUpSample/ISHPullUpSample/WebViewController.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+import UIKit
+import WebKit
+
+class WebViewController: UIViewController {
+    private weak var webView: WKWebView?
+    private weak var topLabel: UILabel?
+
+    override func loadView() {
+        view = ISHPullUpRoundedView(frame: UIScreen.main().bounds)
```
Comment (shagedorn) [inline ISHPullUpSample/ISHPullUpSample/WebViewController.swift]: Comment that this is a hack

File: ISHPullUpSample/ISHPullUpSample/WebViewController.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        label.frame = CGRect(x: 0, y: 0, width: view.bounds.size.width, height: labelHeight)
+        label.autoresizingMask = [.flexibleBottomMargin, .flexibleWidth]
+        label.font = UIFont.preferredFont(forTextStyle: UIFontTextStyleFootnote)
+        label.backgroundColor = .clear()
+        label.textAlignment = .center
+        view.addSubview(label)
+        topLabel = label
+    }
+
+    func loadURL(_ url: URL) {
```
Comment (shagedorn) [inline ISHPullUpSample/ISHPullUpSample/WebViewController.swift]: private

**Comment by shagedorn** [comment]
Only minor remarks. 👍

File: ISHPullUpSample/ISHPullUpSample/WebViewController.swift (hunk 3 above)
Comment (felixLam) [inline ISHPullUpSample/ISHPullUpSample/WebViewController.swift]: This isn't a hack. We provide a custom backing view for the view controller. Will add a comment why we do that.

File: ISHPullUpSample/ISHPullUpSample/WebViewController.swift (hunk 4 above)
Comment (felixLam) [inline ISHPullUpSample/ISHPullUpSample/WebViewController.swift]: This is the only public method and is called externally.

File: ISHPullUpSample/ISHPullUpSample/WebViewController.swift (hunk 3 above)
Comment (shagedorn) [inline ISHPullUpSample/ISHPullUpSample/WebViewController.swift]: Ah. I thought this view only lives temporarily.

File: ISHPullUpSample/ISHPullUpSample/WebViewController.swift (hunk 4 above)
Comment (shagedorn) [inline ISHPullUpSample/ISHPullUpSample/WebViewController.swift]: Sorry... so true. Although it's `internal`, not `public` 😜

**Pull request merged**